media/css/base/footer-refresh.scss

```diff
@@ -0,0 +1,569 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+$color-primary-gray: #181818;
+$color-secondary-gray: #757575;
+$color-tertiary-gray: #828282;
+$color-zilla-green: #00d230;
+$moz24-space-xl: 64px;
+$moz24-space-lg: 24px;
+$moz24-space-gutter: 32px;
+$moz24-space-grid-margin: 36px;
+$moz24-font-size-sm: 12px;
+$moz24-font-size-md: 16px;
```

Comment:
**Question:** might it be worth putting reusable tokens into an include so other files can use them as we move forward with more elements / pages?